### PR TITLE
changed the navbar throughout all pages!

### PR DESCRIPTION
**Pull request by pragathish07**
I have changed all navbar as same as in home page throughout all the pages as metioned , I followed all the instructions mentioned, the code is well structured without disturbing the previous code

Hope this changes helps :) @TheGuyDangerous

### CODE REVIEW

**Comment by pragathish07** [comment]
before 
Screenshot (3)

after adding the same navbar
Screenshot (6)

File: html-files/pages/Page10.html [hunk 1]
```diff
@@ -30,72 +31,29 @@
...
+          </div>
+          <div class="toggle">
+              <span class="first"></span> 
+              <span class="middle"></span>
+              <span class="last"></span>
+          </div>
+          <div class="navigation-bar">
+              <ul>
+                  <li class="active"><a href="../../index.html">Home<span class="underline"></span></a></li>
+                  <li><a href="./html-files/pages/Page1.html">Recipes<span class="underline"></span></a></li>
```
Comment (TheGuyDangerous) [inline html-files/pages/Page10.html]: use ```./Page1.html``` instead. This error is causing the site to have wrong redirection to all the pages Mentioned below -
- Page 2
- Page 4
- Page 5
- Page 6
- Page 7
- Page 8
- Page 9
- Page 10
- Page 11
- Page 12
- Page 13

when the user taps on ```Recipes``` button in the Navbar

File: html-files/pages/Page10.html [hunk 2]
```diff
@@ -30,72 +31,29 @@
...
-                    </div>
-                    <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
-                </div>
-            </nav>
-        </div>
-    </header>
+    <header>
+        <nav>
+          <div class="logo">
+              <a href="index.html"><img height="50" src="https://ik.imagekit.io/daringbaba/Food_Coders/Screenshot_2022-12-23_160309-removebg-preview.png?ik-sdk-version=javascript-1.4.3&updatedAt=1671792391543" alt="Rosa Logo"></a>
```
Comment (TheGuyDangerous) [inline html-files/pages/Page10.html]: at ```<a href="index.html"><img height="50"``` on line 36/37

use ```/index.html``` instead. 

This error is causing false redirection to a page that don't exist.

image


- The following error is happening on all the pages. Fix it, send in an update(do not close the PR, as sending in another PR through your forked repo, will update your changes here) and you'll be good to go ✅

**Review by TheGuyDangerous — CHANGES_REQUESTED** [review]

**Comment by TheGuyDangerous** [comment]
Hi @pragathish07 fix the code review issues, and you'll be good to go ✅

**Review by TheGuyDangerous — CHANGES_REQUESTED** [review]
Fix the media content overflow issue in the navbar, when opened on a mobile device screen.
- A reference for the same is attached below -
Mobile ss

- Also , update the newer pages, by fetching the latest changes made , since your last commit.

**Pull request closed without merging**